ethicml/algorithms/inprocess/fair_dummies.py

```diff
@@ -52,7 +52,9 @@ def _get_path_to_script(self) -> List[str]:
 
     @implements(InAlgorithmSubprocess)
     def _get_flags(self) -> FairDummiesArgs:
-        model_type = "deep_model" if self.model_type.lower() == "deep_model" else "linear_model"
+        model_type: Literal["deep_model", "linear_model"] = (
+            "deep_model" if self.model_type.lower() == "deep_model" else "linear_model"
+        )
```

Comment:
BTW, this is only here to satisfy the Literal type. For future work, the Literal should be replaced with an Enum.

Reply:
should be easy to do